### PR TITLE
Change Text Api component to hooks and functions in 2 examples

### PR DESCRIPTION
**Pull request by luism3861**
i change 2 examples text api component first example to hooks and another in a function.

### CODE REVIEW

**Comment by luism3861** [comment]
it's ready thank you.

**Comment by luism3861** [comment]
> Hey thanks for doing this work! But you made this in the versioned docs! Please revert and apply them to the dos found under **/docs/**, not **/website/versioned_docs/..**

it's ready thank you rachel.

**Comment by luism3861** [comment]
> Hey thanks for doing this work! But you made this in the versioned docs! Please revert and apply them to the dos found under **/docs/**, not **/website/versioned_docs/..**

revert it's ready thanks rachel!!!

File: website/versioned_docs/version-0.60/text.md [hunk 1]
```diff
@@ -14,32 +14,25 @@ In the following example, the nested title and body text will inherit the `fontF
 import React, { Component } from 'react';
```
Comment (lcelso) [inline website/versioned_docs/version-0.60/text.md]: The file correct for changes is 'docs/text.md'

Comment (luism3861) [reply website/versioned_docs/version-0.60/text.md]: yes in the last commit i change this.

Comment (luism3861) [reply website/versioned_docs/version-0.60/text.md]: https://github.com/facebook/react-native-website/pull/1617/files

**Comment by rachelnabors** [comment]
Unfortunately, @lcelso is right. Someone already tackled this, making your PR redundant. Please have a look at #1579 and pick a different one! I'll put your name in the matrix as having claimed it! Thank you for your contribution! Next time!
mlp_ur-the-best

**Pull request closed without merging**